fvtest/porttest/testHelpers.cpp

```diff
@@ -240,18 +240,16 @@ outputErrorMessage(struct OMRPortLibrary *portLibrary, const char *fileName, int
 	va_start(args, format);
```

Comment:
Not sure I understand your question. The va_start() and va_end() calls are made only by variadic methods, which encapsulate variable arg list in va_list that is then passed down the call chain until some receiving method needs to access the contained vars. The receiving method must call va_copy() and iterate over the copy to obtain vars, leaving the original alone.